pkg/daemon/cleanup/cleanup.go

```diff
@@ -69,8 +70,12 @@ func Deconfigure(cfg *rest.Config, ctx context.Context, configMapNamespace strin
 	if err != nil {
 		return err
 	}
-	// rebooting instance to unset the environment variables at the process level as expected
-	if envVarsRemoved {
+	certsRemoved, err := certs.Reconcile("")
+	if err != nil {
+		return err
+	}
+	// rebooting instance to unset the environment variables and available certifictes at the process level as expected
+	if envVarsRemoved || certsRemoved {
```

Comment:
This logically belongs in the `[daemon] Trust bundle cert removal`. I had to return to the commit to figure out how the `""` input is handled.